### PR TITLE
Restore python build for AzureEP

### PR DESCRIPTION
**Pull request by RandySheriffH**
Apply a few arguments to restore building of AzureEP in python, for rel-1.15.0

### CODE REVIEW

File: tools/ci_build/github/linux/build_linux_arm64_python_package.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ if [ "$BUILD_DEVICE" == "GPU" ]; then
     ONNXRUNTIME_CUDA_VERSION="11.8"
     BUILD_ARGS+=("--use_cuda" "--use_tensorrt" "--cuda_version=$ONNXRUNTIME_CUDA_VERSION" "--tensorrt_home=/usr" "--cuda_home=/usr/local/cuda-$ONNXRUNTIME_CUDA_VERSION" "--cudnn_home=/usr/local/cuda-$ONNXRUNTIME_CUDA_VERSION" "--cmake_extra_defines" "CMAKE_CUDA_ARCHITECTURES=52;60;61;70;75;80")
 elif [ "$BUILD_DEVICE" == "AZURE" ]; then
-    BUILD_ARGS+=("--use_azure")
+    BUILD_ARGS+=("--use_azure" "--allow_running_as_root")
```
Comment (snnn) [inline tools/ci_build/github/linux/build_linux_arm64_python_package.sh:48]: Please do not use root

File: tools/ci_build/github/linux/run_python_dockerbuild.sh [hunk 2]
```diff
@@ -8,6 +8,10 @@ x) BUILD_EXTR_PAR=${OPTARG};;
 esac
 done
 
+if [ "$BUILD_EXTR_PAR" = "-d AZURE" ]; then
+    export ADDITIONAL_DOCKER_PARAMETER="${ADDITIONAL_DOCKER_PARAMETER} -u root"
```
Comment (snnn) [inline tools/ci_build/github/linux/run_python_dockerbuild.sh:12]: Please do not use root to build any code

Comment (RandySheriffH) [reply tools/ci_build/github/linux/run_python_dockerbuild.sh:12]: Agree - totally we should not use root to build.
Problem is without root we failed at "yum install SSL..." which required by triton client.
The best way to solve the problem is to rebuild a docker image with package installed ahead of time, problem is - do we still have time to do that, and how?

Comment (snnn) [reply tools/ci_build/github/linux/run_python_dockerbuild.sh:12]: Have you tried to add it to: https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh ?

**Pull request closed without merging**